Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

eval doesn't work as expected with quotes on Solaris.

--- libpki-0.8.9/build/ltmain.sh	Tue Nov 10 13:50:30 2015
+++ libpki-0.8.9/build/ltmain.sh	Wed Nov 11 14:43:19 2015
@@ -657,7 +657,7 @@
     }
 
     if ${opt_dry_run-false}; then :; else
-      eval "$my_cmd"
+      eval $my_cmd
       my_status=$?
       if test "$my_status" -eq 0; then :; else
 	eval "(exit $my_status); $my_fail_exp"
